frontend/javascripts/oxalis/view/right-border-tabs/tree_hierarchy_view.tsx

```diff
@@ -284,6 +300,28 @@ class TreeHierarchyView extends React.PureComponent<Props, State> {
     this.props.onDeleteGroup(groupId);
   }
 
+  shuffleTreeGroupColors(groupId: number) {
+    const groupToTreeMap = createGroupToTreesMap(this.props.trees);
+    const groupIdWithSubgroups = getGroupByIdWithSubgroups(this.props.treeGroups, groupId);
+    const shuffleTreeColorActions = groupIdWithSubgroups.flatMap((subGroupId) => {
+      if (subGroupId in groupToTreeMap)
+        return groupToTreeMap[subGroupId].map((tree) => shuffleTreeColorAction(tree.treeId));
+      return [];
+    });
+    this.props.onBatchActions(shuffleTreeColorActions, "SHUFFLE_TREE_COLOR");
+  }
+
+  setTreeGroupColor(groupId: number, color: Vector3) {
+    const groupToTreeMap = createGroupToTreesMap(this.props.trees);
+    const groupIdWithSubgroups = getGroupByIdWithSubgroups(this.props.treeGroups, groupId);
+    const shuffleTreeColorActions = groupIdWithSubgroups.flatMap((subGroupId) => {
+      if (subGroupId in groupToTreeMap)
+        return groupToTreeMap[subGroupId].map((tree) => setTreeColorAction(tree.treeId, color));
+      return [];
+    });
+    this.props.onBatchActions(shuffleTreeColorActions, "SET_TREE_COLOR");
```

Comment:
`shuffleTreeColorActions` should be `setTreeColorActions`

Reply:
Done.